### ISSUE TITLE
Is 'trim' true or false by default?

### ISSUE THREAD

**Issue by alexshpilkin** [w3c/csvw#867]
The trim [property] and [flag] are specified to be `true` by default, while the default dialect in [§5.9] sets the property to `false`. Which is the authoritative one?

[property]: https://w3c.github.io/csvw/metadata/#dialect-trim
[flag]: http://w3c.github.io/csvw/syntax/#dfn-trim
[§5.9]: https://w3c.github.io/csvw/metadata/#dialect-header

**Comment by gkellogg** [comment]
The default is true. That’s what the 5.9 description said when I followed your reference (which is to the ED, not the REC).

**Comment by alexshpilkin** [comment]
Well, yes, I supposes that bugs are to be filed against the latest version, which as far as I understand the ED. But I may have understood the process wrong.

I was referring to the “default dialect description” in §5.9, it just doesn’t have an anchor:
```
{
  "encoding": "utf-8",
  "lineTerminators": ["\r\n", "\n"],
  "quoteChar": "\"",
  "doubleQuote": true,
  "skipRows": 0,
  "commentPrefix": "#",
  "header": true,
  "headerRowCount": 1,
  "delimiter": ",",
  "skipColumns": 0,
  "skipBlankRows": false,
  "skipInitialSpace": false,
  "trim": false
}
```
Note the last object member.

**Comment by RickMoynihan** [comment]
Yes it looks like the "default dialect description" is wrong and needs updated in accordance with what is stated in the definition list above.
